### PR TITLE
Test

### PR DESCRIPTION
**Pull request by yayoamigo**
This is a copy of one of the projects from the microverse trials

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# Code-rewiew-excercise
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```
Comment (yayoamigo) [inline README.md]: Your README is not professional. Remember that Microverse has provided us with the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) to use for all our projects. So, please use it also and then customize it to your project  and make sure all the sections are covered

File: index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
```
Comment (yayoamigo) [inline index.html:6]: Your title is not representative enough. Please make sure to use a title that represents the project you are working on 👍🏻

Comment (yayoamigo) [inline index.html:6]: Good job on adding your CSS as a link, however you didn't included the proper file in your repository

File: index.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  <body>
+    <section id="welcome-section">
+      <h1>David and Birhanu Portfolio</h1>
+    </section>
+    <section  id="projects">
+      <div class="project-tile"> </div>
+      <a href="https://codepen.io/freeCodeCamp/full/zNqgVx"> our repo </a>
+      </section>
+    
+    <script  src="js/index.js"></script>
```
Comment (yayoamigo) [inline index.html:29]: Please use proper indentation so that your code becomes easiest to read 💯

**Review by yayoamigo — COMMENTED** [review]
Good job so far! 👏 👏
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️

Required Changes ♻️

•	[ ] Good job on setting up linters. However, they should all pass(i.e: they are all green) so please make sure that they are all passing 😃 👍

Please check my comments below and make all the required changes .

**Review by frank1738 — APPROVED** [review]
Hi @yayoamigo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

YesYouAreGIF (2) 

## Highlights

✅ Correct use of GitHub inline comments
✅ Provided a reason as to why a change is required

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**